lexicons/com/atproto/sync/subscribeRepos.json

```
@@ -0,0 +1,134 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.sync.subscribeRepos",
+  "defs": {
+    "main": {
+      "type": "subscription",
+      "description": "Subscribe to repo updates",
+      "parameters": {
+        "type": "params",
+        "properties": {
+          "cursor": {
+            "type": "integer",
+            "description": "The last known event to backfill from."
+          }
+        }
+      },
+      "message": {
+        "type": "union",
+        "refs": [
+          "#commit",
+          "#signpost",
+          "#handle",
+          "#migrate",
+          "#tombstone",
+          "#info"
+        ]
+      },
+      "errors": [
+        {"name": "FutureCursor"}
+      ]
+    },
+    "commit": {
+      "type": "object",
+      "required": ["seq", "event", "repo", "commit", "prev", "blocks", "ops", "blobs", "time"],
+      "nullable": ["prev"],
+      "properties": {
+        "seq": {"type": "integer"},
+        "event": { 
+          "type": "string",
+          "knownValues": [
+            "append",
+            "rebase"
+          ]
+        },
+        "repo": {"type": "string"},
+        "commit": {"type": "string"},
+        "prev": {"type": "string"},
+        "blocks": {"type": "unknown"},
+        "ops": {
+          "type": "array", 
+          "items": { "type": "ref", "ref": "#repoOp"}
+        },
+        "blobs": {
+          "type": "array",
+          "items": {"type": "string"}
+        },
+        "time": {"type": "datetime"}
+      }
+    },
+    "signpost": {
+      "type": "object",
+      "required": ["seq", "event", "repo", "commit", "prev", "time"],
+      "nullable": ["prev"],
+      "properties": {
+        "seq": {"type": "integer"},
+        "event": { 
+          "type": "string",
+          "knownValues": [
+            "append",
+            "rebase"
+          ]
+        },
+        "repo": {"type": "string"},
+        "commit": {"type": "string"},
+        "prev": {"type": "string"},
+        "time": {"type": "datetime"}
+      }
+    },
+    "handle": {
+      "type": "object",
+      "required": ["seq", "did", "handle"],
+      "properties": {
+        "seq": {"type": "integer"},
+        "did": {"type": "string"},
+        "handle": {"type": "string"}
+      }
+    },
+    "migrate": {
+      "type": "object",
+      "required": ["seq", "did", "migrateTo"],
+      "nullable": ["migrateTo"],
+      "properties": {
+        "seq": {"type": "integer"},
+        "did": {"type": "string"},
+        "migrateTo": {"type": "string"}
+      }
+    },
+    "tombstone": {
+      "type": "object",
+      "required": ["seq", "did"],
+      "properties": {
+        "seq": {"type": "integer"},
+        "did": {"type": "string"}
+      }
+    },
+    "info": {
+      "type": "object",
+      "required": ["name"],
+      "properties": {
+        "name": {
+          "type": "string",
+          "knownValues": ["OutdatedCursor"]
+        }
```

The info frames also have/had a `message` field if we want to include that. 
```suggestion
        "name": {
          "type": "string",
          "knownValues": ["OutdatedCursor"]
        },
        "message": {
          "type": "string"
        }
```